ddsp/training/train_util.py

```diff
@@ -134,7 +141,9 @@ def train(data_provider,
           steps_per_summary=300,
           steps_per_save=300,
           save_dir='~/tmp/ddsp',
-          restore_dir='~/tmp/ddsp'):
+          restore_dir='~/tmp/ddsp',
+          skip_writing=False,
```

Comment:
I think it would be cleaner to not have `skip_writing` but rather just check for `if save_dir is not None` and then pass in None for save_dir for the other workers that aren't the chief.

Reply:
Yes, it seems to be a better approach. To make it possible I had to use a [mock context manager](https://www.tensorflow.org/api_docs/python/tf/summary/create_noop_writer) as `summary_writer` needs to be defined and it uses `save_dir`. Is it okay?

Reply:
Yah this looks good. It would be great to add a docstring to this function if you could.

Reply:
Done